aten/src/ATen/cpu/vec/vec256/vec256_mask.h

```diff
@@ -31,27 +32,187 @@ struct VecMaskLoad<
   }
 };
 
+template <typename T, typename mask_t, int mask_n>
+struct VecMaskLoad<
+    T,
+    2,
+    mask_t,
+    mask_n,
+    typename std::enable_if_t<
+        (mask_n == 2 || mask_n == 4) &&
+            (std::is_same_v<T, float> || std::is_same_v<T, int32_t> ||
+             std::is_same_v<T, uint32_t>),
+        void>> {
```

Comment:
Can we use "N" and "2*N" to generalize this?

Reply:
Revised according to suggestions.